src/loaders/lottie/tvgLottieLoader.h

```diff
@@ -29,7 +29,7 @@
 
 struct LottieComposition;
 struct LottieBuilder;
-
+enum SelectedSegment{NONE,MARKER,RANGE};
```

Comment:
The enumeration is only needed with marker. Maybe we can define them in LottieMarker?
You can refer to the other model defination in `tvgLottieModel.h`
```cpp
struct LottieMarker
{
    enum Segment : uint8_t { None = 0, Marker, Range };

    char* name = nullptr;
    int time = 0;
    int duration = 0;

    ~LottieMarker()
    {
        free(name);
    }
};
```